java/kafka/consumer/src/main/java/KafkaConsumerConfig.java

```diff
@@ -56,7 +76,15 @@ public KafkaConsumerConfig(String bootstrapServers, String topic, String groupId
         this.additionalConfig = additionalConfig;
     }
 
-    public static KafkaConsumerConfig fromEnv() {
+    public static KafkaConsumerConfig fromEnv() { // Populate JAEGER_SERVICE_NAME and OTEL_SERVICE_NAME with User defined JAEGER_SERVICE_NAME
+        // because OpeTracing uses JAEGER_SERVICE_NAME and OpenTelemetry uses OTEL_SERVICE_NAME.
+        String serviceName = System.getenv("OTEL_SERVICE_NAME")  == null ? DEFAULT_SERVICE_NAME : System.getenv("OTEL_SERVICE_NAME");
+        String tracesExporterNames = System.getenv("OTEL_TRACES_EXPORTER")  == null ? DEFAULT_EXPORTER_NAMES : System.getenv("OTEL_TRACES_EXPORTER");
+        String metricsExporterNames = System.getenv("OTEL_METRICS_EXPORTER")  == null ? DEFAULT_METRICS_NAMES : System.getenv("OTEL_METRICS_EXPORTER");
+
+        boolean openTracingEnabled = System.getenv("OPENTRACING_ENABLED")  == null ? DEFAULT_OPENTRACING_ENABLED : Boolean.parseBoolean(System.getenv("OPENTRACING_ENABLED"));
+        boolean openTelemetryEnabled = System.getenv("OPENTELEMETRY_ENABLED")  == null ? DEFAULT_OPENTELEMETRY_ENABLED : Boolean.parseBoolean(System.getenv("OPENTELEMETRY_ENABLED"));
```

Comment:
see the comment above. I think we need a `tracingSystem` variable not two of them for enabling one or the other.